app/types/app_version_type.rb

```diff
@@ -0,0 +1,15 @@
+class AppVersionType < ActiveRecord::Type::Value
+  def cast(value)
+    AppVersion.new(value) unless value.nil?
+  rescue StandardError
+    nil
+  end
+
+  def serialize(value)
+    value.to_pg_array if value
```

Comment:
Rubocop didn't make you use safe navigation?

Reply:
TT is not on the latest house stuff prolly.